src/lib389/lib389/cli_conf/plugins/ldappassthrough.py

```diff
@@ -88,6 +88,7 @@ def pta_add(inst, basedn, log, args):
     if new_url_l in urls:
         raise ldap.ALREADY_EXISTS("Entry %s already exists" % args.URL)
     plugin.add("nsslapd-pluginarg%s" % next_num, args.URL)
+    log.info("Command successful.")
```

Comment:
I think this should log something like this: "Successfully added URL".  This is how most of the other "success" messages are logged  "Command successful" is not used any where else, so we should try and be consistent with other CLI  success messages.

But otherwise looks good!  Ack